src/Components/Energy/EnergyButton.css

```diff
@@ -1,7 +1,6 @@
 #energyButton {
-  width: 160px;
-  height: 40px;
+  width: 158px;
+  height: 38px;
```

Comment:
please explain this change, either in comment or in code

Reply:
I know what's the problem here, and yet decided it is not worth the time fixing, since it will be deleted soon.